readthedocs/proxito/views/mixins.py

```diff
@@ -129,13 +129,15 @@ def _serve_docs_nginx(self, request, final_project, version_slug, path, download
         # Include the project & project-version so we can do larger purges if needed
         response['Cache-Tags'] = f'{final_project.slug}-{version_slug},{final_project.slug}'
         if hasattr(request, 'rtdheader'):
-            response['X-RTD-Version-Method'] = 'rtdheader'
+            response['X-RTD-Project-Method'] = 'rtdheader'
         if hasattr(request, 'subdomain'):
-            response['X-RTD-Version-Method'] = 'subdomain'
+            response['X-RTD-Project-Method'] = 'subdomain'
+        if hasattr(request, 'cname'):
+            response['X-RTD-Project-Method'] = 'cname'
         if hasattr(request, 'external_domain'):
             response['X-RTD-Version-Method'] = 'external_domain'
-        if hasattr(request, 'cname'):
-            response['X-RTD-Version-Method'] = 'cname'
+        else:
+            response['X-RTD-Version-Method'] = 'url'
```

Comment:
Let's remove this logic entirely and only send it on `external_domain`, that removes a lot of my issue with it.

Reply:
If we don't send this header from Proxito, how would be the NGINX config? Do we need to add an `if` there to avoid checking for this variable?

Reply:
I'm not sure. I guess this is fine for now if we care, but I think for PR's, we know where the version came from, so I don't know if this is super useful. 🤷